codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AddS3Config.java

```diff
@@ -178,14 +180,13 @@ public void addConfigInterfaceFields(TypeScriptSettings settings, Model model, S
                         .build(),
                 RuntimeClientPlugin.builder()
                         .withConventions(AwsDependency.SSEC_MIDDLEWARE.dependency, "Ssec", HAS_MIDDLEWARE)
-                        .operationPredicate((m, s, o) -> testInputContainsMember(m, o, SSEC_OPERATIONS)
+                        .operationPredicate((m, s, o) -> testInputContainsMember(m, o, SSEC_INPUT_KEYS)
                                             && testServiceId(s))
                         .build(),
                 RuntimeClientPlugin.builder()
                         .withConventions(AwsDependency.LOCATION_CONSTRAINT.dependency, "LocationConstraint",
                                          HAS_MIDDLEWARE)
-                        .operationPredicate((m, s, o) -> o.getId().getName(s).equals("CreateBucket")
-                                            && testServiceId(s))
+                        .operationPredicate((m, s, o) -> o.getId().getName(s).equals("CreateBucket") && testServiceId(s))
```

Comment:
fix for checkstyle error
```suggestion
                        .operationPredicate((m, s, o) -> o.getId().getName(s).equals("CreateBucket")
                                && testServiceId(s))
```